person.rb

```diff
@@ -18,6 +18,14 @@ def can_use_services?
     of_age || @parent_permission
   end
 
+  def correct_name
+    decorated_name(@name)
+  end
+
+  def add_rental(date, book)
+    @rentals << Rental.new(book, self, date)
+  end
```

Comment:
- [ ] Good job adding this rentals method here 👍 but you also have to add `@rentals = []` to the initialize method of person so as to initialize the rentals array. 
- [ ] Also, kindly add rentals to the `attr_accessor` of the Person class too 💯